From e3490e08014d52de9cecb7ff4917a7579940262f Mon Sep 17 00:00:00 2001
From: Bill Yi <byi@google.com>
Date: Thu, 28 Apr 2022 18:46:21 +0000
Subject: [PATCH] Import translations. DO NOT MERGE ANYWHERE

BUG:205460459

Auto-generated-cl: translation import
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:aedbfeb8b85eff1f28fcaaf71d0826f76f7afab7)
Merged-In: I5bb2a5d970bbdff0ea715cef8f0fca7bd052a670
Change-Id: I5bb2a5d970bbdff0ea715cef8f0fca7bd052a670
---
 res/values-af/strings.xml        | 2 +-
 res/values-am/strings.xml        | 2 +-
 res/values-ar/strings.xml        | 2 +-
 res/values-as/strings.xml        | 2 +-
 res/values-az/strings.xml        | 2 +-
 res/values-b+sr+Latn/strings.xml | 2 +-
 res/values-be/strings.xml        | 2 +-
 res/values-bg/strings.xml        | 2 +-
 res/values-bn/strings.xml        | 2 +-
 res/values-bs/strings.xml        | 2 +-
 res/values-ca/strings.xml        | 2 +-
 res/values-cs/strings.xml        | 2 +-
 res/values-da/strings.xml        | 2 +-
 res/values-de/strings.xml        | 2 +-
 res/values-el/strings.xml        | 2 +-
 res/values-en-rAU/strings.xml    | 2 +-
 res/values-en-rCA/strings.xml    | 2 +-
 res/values-en-rGB/strings.xml    | 2 +-
 res/values-en-rIN/strings.xml    | 2 +-
 res/values-en-rXC/strings.xml    | 2 +-
 res/values-es-rUS/strings.xml    | 2 +-
 res/values-es/strings.xml        | 2 +-
 res/values-et/strings.xml        | 2 +-
 res/values-eu/strings.xml        | 2 +-
 res/values-fa/strings.xml        | 2 +-
 res/values-fi/strings.xml        | 2 +-
 res/values-fr-rCA/strings.xml    | 2 +-
 res/values-fr/strings.xml        | 2 +-
 res/values-gl/strings.xml        | 2 +-
 res/values-gu/strings.xml        | 2 +-
 res/values-hi/strings.xml        | 2 +-
 res/values-hr/strings.xml        | 2 +-
 res/values-hu/strings.xml        | 2 +-
 res/values-hy/strings.xml        | 2 +-
 res/values-in/strings.xml        | 2 +-
 res/values-is/strings.xml        | 2 +-
 res/values-it/strings.xml        | 2 +-
 res/values-iw/strings.xml        | 2 +-
 res/values-ja/strings.xml        | 2 +-
 res/values-ka/strings.xml        | 2 +-
 res/values-kk/strings.xml        | 2 +-
 res/values-km/strings.xml        | 2 +-
 res/values-kn/strings.xml        | 2 +-
 res/values-ko/strings.xml        | 2 +-
 res/values-ky/strings.xml        | 2 +-
 res/values-lo/strings.xml        | 2 +-
 res/values-lt/strings.xml        | 2 +-
 res/values-lv/strings.xml        | 2 +-
 res/values-mk/strings.xml        | 2 +-
 res/values-ml/strings.xml        | 2 +-
 res/values-mn/strings.xml        | 2 +-
 res/values-mr/strings.xml        | 2 +-
 res/values-ms/strings.xml        | 2 +-
 res/values-my/strings.xml        | 2 +-
 res/values-nb/strings.xml        | 2 +-
 res/values-ne/strings.xml        | 2 +-
 res/values-nl/strings.xml        | 2 +-
 res/values-or/strings.xml        | 2 +-
 res/values-pa/strings.xml        | 2 +-
 res/values-pl/strings.xml        | 2 +-
 res/values-pt-rBR/strings.xml    | 2 +-
 res/values-pt-rPT/strings.xml    | 2 +-
 res/values-pt/strings.xml        | 2 +-
 res/values-ro/strings.xml        | 2 +-
 res/values-ru/strings.xml        | 2 +-
 res/values-si/strings.xml        | 2 +-
 res/values-sk/strings.xml        | 2 +-
 res/values-sl/strings.xml        | 2 +-
 res/values-sq/strings.xml        | 2 +-
 res/values-sr/strings.xml        | 2 +-
 res/values-sv/strings.xml        | 2 +-
 res/values-sw/strings.xml        | 2 +-
 res/values-ta/strings.xml        | 2 +-
 res/values-te/strings.xml        | 2 +-
 res/values-th/strings.xml        | 2 +-
 res/values-tl/strings.xml        | 2 +-
 res/values-tr/strings.xml        | 2 +-
 res/values-uk/strings.xml        | 2 +-
 res/values-ur/strings.xml        | 2 +-
 res/values-uz/strings.xml        | 2 +-
 res/values-vi/strings.xml        | 2 +-
 res/values-zh-rCN/strings.xml    | 2 +-
 res/values-zh-rHK/strings.xml    | 2 +-
 res/values-zh-rTW/strings.xml    | 2 +-
 res/values-zu/strings.xml        | 2 +-
 85 files changed, 85 insertions(+), 85 deletions(-)

diff --git a/res/values-af/strings.xml b/res/values-af/strings.xml
index 73096d719a..ed3b78f3e1 100644
--- a/res/values-af/strings.xml
+++ b/res/values-af/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Koppel aan <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Ontkoppel hierdie VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Ontkoppel"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Weergawe <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Weergawe"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Vergeet VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Vervang bestaande VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Stel altyd-aan VPN?"</string>
diff --git a/res/values-am/strings.xml b/res/values-am/strings.xml
index aaac29e803..59e8bae505 100644
--- a/res/values-am/strings.xml
+++ b/res/values-am/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"ከ<xliff:g id="PROFILE">%s</xliff:g> ጋር ተገናኝ"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"የዚህ ቪፒኤን ግንኙነት ይቋረጥ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ግንኙነት አቋርጥ"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ሥሪት <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ስሪት"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPNን እርሳ"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"አሁን ያለው VPN ይተካ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ሁልጊዜ-የበራ VPN ይቀናበር?"</string>
diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 47d6130936..fec8fe94d0 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -3389,7 +3389,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"الاتصال بـ <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"‏هل تريد قطع الاتصال بهذه الشبكة الافتراضية الخاصة (VPN)؟"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"قطع الاتصال"</string>
-    <string name="vpn_version" msgid="41856553718304367">"الإصدار <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"الإصدار"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"نسيان الشبكة الافتراضية الخاصة"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"هل تريد استبدال الشبكة الافتراضية الخاصة الحالية؟"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"هل تريد تشغيل الشبكة الافتراضية الخاصة باستمرار؟"</string>
diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index 5b65255f94..539bed2f20 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g>ৰ সৈতে সংযোগ কৰক"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"এই ভিপিএনটোৰ সৈতে সংযোগ বিচ্ছিন্ন কৰিবনে?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"সংযোগ বিচ্ছিন্ন কৰক"</string>
-    <string name="vpn_version" msgid="41856553718304367">"সংস্কৰণ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"সংস্কৰণ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"ভিপিএন পাহৰক"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"পূৰ্বৰ ভিপিএনটো সলনি কৰেনে?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ভিপিএনক যিকোনো সময়ত অন থকা অবস্থাত ছেট কৰিবনে?"</string>
diff --git a/res/values-az/strings.xml b/res/values-az/strings.xml
index c3fa780370..b0a1303421 100644
--- a/res/values-az/strings.xml
+++ b/res/values-az/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> profilinə qoşulun"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Bu VPN bağlantısı kəsilsin?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Bağlantını kəsin"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versiya <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versiya"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN\'i unudun"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Mövcud VPN əvəz edilsin?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Hər zaman aktiv VPN quraşdırılsın?"</string>
diff --git a/res/values-b+sr+Latn/strings.xml b/res/values-b+sr+Latn/strings.xml
index e8800ae80d..a498165852 100644
--- a/res/values-b+sr+Latn/strings.xml
+++ b/res/values-b+sr+Latn/strings.xml
@@ -3269,7 +3269,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Povezivanje na profil <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Želite li da prekinete vezu sa ovim VPN-om?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Prekini vezu"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verzija <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verzija"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zaboravi VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Želite li da zamenite postojeći VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Želite li da podesite uvek uključeni VPN?"</string>
diff --git a/res/values-be/strings.xml b/res/values-be/strings.xml
index 71a13253fb..2192455966 100644
--- a/res/values-be/strings.xml
+++ b/res/values-be/strings.xml
@@ -3311,7 +3311,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Падключэнне да профiлю <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Адключыць гэту VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Адключыць"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Версія <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Версія"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Забыць VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Замяніць існуючую VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Наладзіць заўсёды ўключаную VPN?"</string>
diff --git a/res/values-bg/strings.xml b/res/values-bg/strings.xml
index 0f13d09d71..e009ed4940 100644
--- a/res/values-bg/strings.xml
+++ b/res/values-bg/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Свързване със: <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Да се прекрати ли връзката с тази виртуална частна мрежа (VPN)?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Прекратяване на връзката"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Версия <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Версия"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Забравяне на вирт. частна мрежа (VPN)"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Да се замени ли съществуващата VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Да се зададе ли винаги включена виртуална частна мрежа (VPN)?"</string>
diff --git a/res/values-bn/strings.xml b/res/values-bn/strings.xml
index 491a1507d1..0abacd9312 100644
--- a/res/values-bn/strings.xml
+++ b/res/values-bn/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> এতে কানেক্ট করুন"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"এই VPN থেকে ডিসকানেক্ট করবেন?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ডিসকানেক্ট করুন"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ভার্সন <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ভার্সন"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN সরিয়ে দিন"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"বিদ্যমান VPN প্রতিস্থাপন করতে চান?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"সর্বদা-চালু VPN সেট করতে চান?"</string>
diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index b4e35c2203..c110f480c3 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -3269,7 +3269,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Poveži se na <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Želite li prekinuti vezu s ovim VPN-om?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Prekini vezu"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verzija <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verzija"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zaboravi VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Želite li zamijeniti postojeći VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Želite li postaviti uvijek uključen VPN?"</string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index 832ae14db8..ae5c36fc08 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connexió amb <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Vols desconnectar aquesta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconnecta"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versió <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versió"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Oblida la VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Vols substituir la VPN actual?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Vols definir la VPN sempre activa?"</string>
diff --git a/res/values-cs/strings.xml b/res/values-cs/strings.xml
index e3ccef4806..149113ad88 100644
--- a/res/values-cs/strings.xml
+++ b/res/values-cs/strings.xml
@@ -3309,7 +3309,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Připojit k profilu <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Odpojit tuto síť VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Odpojit"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verze <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verze"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zapomenout VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Nahradit stávající síť VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Nastavit trvalou síť VPN?"</string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index 3d03f0363e..76cd2977ed 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Opret forbindelse til <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Vil du afbryde dette VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Afbryd"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Glem VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Erstat det eksisterende VPN-netværk?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Vil du konfigurere konstant VPN?"</string>
diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index 28dbe6199d..087ba09bcd 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -3232,7 +3232,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Mit <xliff:g id="PROFILE">%s</xliff:g> verbinden"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Verbindung zu diesem VPN trennen?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Verbindung trennen"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN entfernen"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Aktuelles VPN ersetzen?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Durchgehend aktives VPN aktivieren?"</string>
diff --git a/res/values-el/strings.xml b/res/values-el/strings.xml
index c002baba71..2732779018 100644
--- a/res/values-el/strings.xml
+++ b/res/values-el/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Σύνδεση στο προφίλ <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Να αποσυνδεθεί αυτό το VPN;"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Αποσύνδεση"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Έκδοση <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Έκδοση"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Διαγραφή VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Αντικατάσταση υπάρχοντος VPN;"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Ενεργοποίηση του μονίμως ενεργού VPN;"</string>
diff --git a/res/values-en-rAU/strings.xml b/res/values-en-rAU/strings.xml
index 414cea4327..d75488f532 100644
--- a/res/values-en-rAU/strings.xml
+++ b/res/values-en-rAU/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connect to <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Disconnect this VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Disconnect"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Forget VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Replace existing VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Set always-on VPN?"</string>
diff --git a/res/values-en-rCA/strings.xml b/res/values-en-rCA/strings.xml
index f998aa81bb..f99d1a72d9 100644
--- a/res/values-en-rCA/strings.xml
+++ b/res/values-en-rCA/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connect to <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Disconnect this VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Disconnect"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Forget VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Replace existing VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Set always-on VPN?"</string>
diff --git a/res/values-en-rGB/strings.xml b/res/values-en-rGB/strings.xml
index d46d4f9d8a..49369cd928 100644
--- a/res/values-en-rGB/strings.xml
+++ b/res/values-en-rGB/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connect to <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Disconnect this VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Disconnect"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Forget VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Replace existing VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Set always-on VPN?"</string>
diff --git a/res/values-en-rIN/strings.xml b/res/values-en-rIN/strings.xml
index a19e681997..d9badac277 100644
--- a/res/values-en-rIN/strings.xml
+++ b/res/values-en-rIN/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connect to <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Disconnect this VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Disconnect"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Forget VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Replace existing VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Set always-on VPN?"</string>
diff --git a/res/values-en-rXC/strings.xml b/res/values-en-rXC/strings.xml
index b2f7f985f7..a7bd40dc52 100644
--- a/res/values-en-rXC/strings.xml
+++ b/res/values-en-rXC/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‎‏‎‏‏‎‎‎‎‎‎‎‏‏‏‏‎‎‏‏‏‏‏‏‎‏‏‎‎‏‏‏‎‎‎‏‎‏‎‏‎‎‎‏‎‎‏‏‎‎‎‏‏‎‎‎‏‏‎‏‎‎Connect to ‎‏‎‎‏‏‎<xliff:g id="PROFILE">%s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‎"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‎‎‎‎‎‏‏‎‏‏‏‏‏‎‏‏‎‏‎‎‎‏‏‏‎‏‏‏‎‏‎‏‎‎‎‎‎‏‏‎‏‏‏‎‎‎‏‏‏‏‎‎‎‎‎‏‎‏‏‎Disconnect this VPN?‎‏‎‎‏‎"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‏‏‎‎‏‏‎‏‏‎‎‎‏‎‎‎‎‎‎‏‎‏‎‏‏‎‎‏‎‏‏‎‏‏‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‎‏‎‏‎‎‎‏‎Disconnect‎‏‎‎‏‎"</string>
-    <string name="vpn_version" msgid="41856553718304367">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‎‎‎‏‎‎‏‎‏‎‎‏‎‏‏‎‏‎‎‎‏‎‏‎‎‎‎‎‏‎‏‎‏‏‏‏‏‏‎‎‏‏‎‏‏‎‎‏‏‏‎‎‏‏‎‏‏‏‏‎Version ‎‏‎‎‏‏‎<xliff:g id="VERSION">%s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‎"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‎‎‎‎‎‎‎‏‎‏‏‎‎‎‎‎‎‏‎‎‏‏‏‎‎‏‎‏‏‎‎‎‎‎‏‏‏‏‎‏‎‏‎‏‏‏‎‏‎‏‎‎‏‎‏‎‎‎‎‎Version‎‏‎‎‏‎"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‎‏‎‎‎‎‏‏‏‏‎‎‏‏‎‎‏‎‏‏‏‏‎‏‏‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‎‏‏‏‏‎‏‎‎‎‏‏‏‏‏‏‏‎‏‎Forget VPN‎‏‎‎‏‎"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‎‏‏‎‏‏‏‎‏‏‎‏‏‏‎‎‏‎‏‎‏‏‏‏‏‏‎‎‏‎‏‎‏‎‎‏‏‎‎‎‏‏‏‏‎‎‏‎‎‎‏‎‏‏‏‏‎‏‏‏‎‎‎‎Replace existing VPN?‎‏‎‎‏‎"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‏‏‎‏‏‏‎‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‎‏‎‏‎‎‏‏‎‎‏‎‎‏‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‎‏‎‎Set always-on VPN?‎‏‎‎‏‎"</string>
diff --git a/res/values-es-rUS/strings.xml b/res/values-es-rUS/strings.xml
index 1e4949f40d..6bc43a039d 100644
--- a/res/values-es-rUS/strings.xml
+++ b/res/values-es-rUS/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectar a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"¿Deseas desconectar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconectar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versión <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versión"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Borrar VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"¿Reemplazar VPN existente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"¿Configurar la VPN siempre activa?"</string>
diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index 7bf847fe63..c7c40c923f 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectar a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"¿Desconectar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconectar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versión <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versión"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Olvidar VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"¿Reemplazar VPN actual?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"¿Configurar el modo de VPN siempre activada?"</string>
diff --git a/res/values-et/strings.xml b/res/values-et/strings.xml
index 612adccad1..91f6f927ad 100644
--- a/res/values-et/strings.xml
+++ b/res/values-et/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Profiiliga <xliff:g id="PROFILE">%s</xliff:g> ühendamine"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Kas soovite katkestada ühenduse selle VPN-iga?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Katkesta ühendus"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versioon <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versioon"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Unusta VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Kas asendada olemasolev VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Kas määrata alati sisselülitatud VPN?"</string>
diff --git a/res/values-eu/strings.xml b/res/values-eu/strings.xml
index 8a86629983..b09578a67e 100644
--- a/res/values-eu/strings.xml
+++ b/res/values-eu/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Konektatu <xliff:g id="PROFILE">%s</xliff:g> profilera"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"VPN saretik deskonektatu nahi duzu?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Deskonektatu"</string>
-    <string name="vpn_version" msgid="41856553718304367">"<xliff:g id="VERSION">%s</xliff:g> bertsioa"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Bertsioa"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Ahaztu VPN konexioa"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Dagoen VPN konexioa ordeztu nahi duzu?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"VPNa beti aktibatuta egoteko aukera ezarri nahi duzu?"</string>
diff --git a/res/values-fa/strings.xml b/res/values-fa/strings.xml
index e387d00e36..bcf1edc7a4 100644
--- a/res/values-fa/strings.xml
+++ b/res/values-fa/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"اتصال به <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"‏اتصال به این VPN.قطع‌ شود؟"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"قطع ارتباط"</string>
-    <string name="vpn_version" msgid="41856553718304367">"نسخه <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"نسخه"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"‏فراموش کردن VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"‏VPN موجود جایگزین شود؟"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"‏VPN همیشه روشن تنظیم شود؟"</string>
diff --git a/res/values-fi/strings.xml b/res/values-fi/strings.xml
index 959aa52d61..b4d95fb296 100644
--- a/res/values-fi/strings.xml
+++ b/res/values-fi/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Yhdistä profiiliin <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Katkaistaanko VPN-yhteys?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Katkaise yhteys"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versio <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versio"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Unohda VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Korvataanko nykyinen VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Määritetäänkö aina käytössä oleva VPN?"</string>
diff --git a/res/values-fr-rCA/strings.xml b/res/values-fr-rCA/strings.xml
index 000af661ba..f44531df92 100644
--- a/res/values-fr-rCA/strings.xml
+++ b/res/values-fr-rCA/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Se connecter à <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Déconnecter ce RPV?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Déconnecter"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Oublier le profil RPV"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Remplacer le RPV existant?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Définir le RPV permanent?"</string>
diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index bebc758574..a023498454 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Se connecter à <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Déconnecter ce VPN ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Déconnecter"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Supprimer le VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Remplacer le VPN existant ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Définir le VPN permanent ?"</string>
diff --git a/res/values-gl/strings.xml b/res/values-gl/strings.xml
index 9725d1a8cc..0adcdae9e8 100644
--- a/res/values-gl/strings.xml
+++ b/res/values-gl/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectar con <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Queres desconectar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconectar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versión <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versión"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Esquecer perfil da VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Queres substituír a VPN existente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Queres definir a VPN como sempre activa?"</string>
diff --git a/res/values-gu/strings.xml b/res/values-gu/strings.xml
index 3cdfca79bd..04215f2b4e 100644
--- a/res/values-gu/strings.xml
+++ b/res/values-gu/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> સાથે કનેક્ટ કરો"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"આ VPNને ડિસ્કનેક્ટ કરીએ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ડિસ્કનેક્ટ કરો"</string>
-    <string name="vpn_version" msgid="41856553718304367">"વર્ઝન <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"વર્ઝન"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN ભૂલી ગયાં"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"અસ્તિત્વમાંની VPN ને બદલીએ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"હંમેશાં ચાલુ VPN સેટ કરીએ?"</string>
diff --git a/res/values-hi/strings.xml b/res/values-hi/strings.xml
index 3e9058b911..2350483052 100644
--- a/res/values-hi/strings.xml
+++ b/res/values-hi/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">" <xliff:g id="PROFILE">%s</xliff:g> से कनेक्ट करें"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"इस VPN को डिसकनेक्ट करना चाहते हैं?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"डिसकनेक्ट करें"</string>
-    <string name="vpn_version" msgid="41856553718304367">"वर्शन <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"वर्शन"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"वीपीएन भूल जाएं"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"मौजूदा VPN को बदलें?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"हमेशा-चालू VPN सेट करें?"</string>
diff --git a/res/values-hr/strings.xml b/res/values-hr/strings.xml
index e2d9c81ee0..57c5c964e4 100644
--- a/res/values-hr/strings.xml
+++ b/res/values-hr/strings.xml
@@ -3271,7 +3271,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Povezivanje s profilom <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Prekinuti vezu s VPN-om?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Prekini vezu"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verzija <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verzija"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zaboravi VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Želite li zamijeniti dosadašnji VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Želite li postaviti uvijek uključeni VPN?"</string>
diff --git a/res/values-hu/strings.xml b/res/values-hu/strings.xml
index 1e2afc27df..b1fd379972 100644
--- a/res/values-hu/strings.xml
+++ b/res/values-hu/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Csatlakozás a következőhöz: <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Leválasztja a VPN-t?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Leválasztás"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verzió: <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verzió"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN elfelejtése"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Lecseréli a meglévő VPN-t?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Beállítja a mindig bekapcsolt VPN-t?"</string>
diff --git a/res/values-hy/strings.xml b/res/values-hy/strings.xml
index 3310712ea4..3337a6644a 100644
--- a/res/values-hy/strings.xml
+++ b/res/values-hy/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Միանալ <xliff:g id="PROFILE">%s</xliff:g>-ին"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Անջատե՞լ այս VPN-ը:"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Անջատել"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Տարբերակ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Տարբերակ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Մոռանալ VPN-ը"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Փոխարինե՞լ ընթացիկ VPN-ը:"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Սահմանե՞լ որպես «միշտ միացված» VPN:"</string>
diff --git a/res/values-in/strings.xml b/res/values-in/strings.xml
index 95fd4923cd..b1670e43e6 100644
--- a/res/values-in/strings.xml
+++ b/res/values-in/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Menghubungkan ke <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Putuskan sambungan VPN ini?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Putuskan koneksi"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versi <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versi"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Lupakan VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Ganti VPN yang sudah ada?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Setel VPN selalu aktif?"</string>
diff --git a/res/values-is/strings.xml b/res/values-is/strings.xml
index cfd733b23d..a329a287c2 100644
--- a/res/values-is/strings.xml
+++ b/res/values-is/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Tengjast <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Aftengja þessa VPN-tengingu?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Aftengja"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Útgáfa <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Útgáfa"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Gleyma VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Skipta út núverandi VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Velja VPN sem alltaf er kveikt á?"</string>
diff --git a/res/values-it/strings.xml b/res/values-it/strings.xml
index fc132ed60c..5c7ece01df 100644
--- a/res/values-it/strings.xml
+++ b/res/values-it/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Connessione a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Disconnettere questa VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Disconnetti"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versione <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versione"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Rimuovi VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Sostituire la rete VPN esistente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Impostare la rete VPN sempre attiva?"</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index 581e690209..224edc15ad 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -3311,7 +3311,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"התחברות אל <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"‏האם לנתק את ה-VPN הזה?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ניתוק"</string>
-    <string name="vpn_version" msgid="41856553718304367">"גרסה <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"גרסה"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"‏אני רוצה לשכוח את ה-VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"‏האם להחליף רשת VPN קיימת?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"‏האם להגדיר את רשת ה-VPN בחיבור תמידי?"</string>
diff --git a/res/values-ja/strings.xml b/res/values-ja/strings.xml
index 7a03c44ac4..dc5a41567e 100644
--- a/res/values-ja/strings.xml
+++ b/res/values-ja/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g>に接続"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"この VPN の接続を解除しますか？"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"接続を解除"</string>
-    <string name="vpn_version" msgid="41856553718304367">"バージョン<xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"バージョン"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN を削除"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"既存の VPN を置き換えますか？"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"常時接続 VPN を設定しますか？"</string>
diff --git a/res/values-ka/strings.xml b/res/values-ka/strings.xml
index 38fad4e737..510eff5377 100644
--- a/res/values-ka/strings.xml
+++ b/res/values-ka/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"დაკავშირებულია <xliff:g id="PROFILE">%s</xliff:g>-თან"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"გსურთ ამ VPN-თან კავშირის გაწყვეტა?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"კავშირის გაწყვეტა"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ვერსია <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ვერსია"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN-ის დავიწყება"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"გსურთ არსებული VPN-ის ჩანაცვლება?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"გსურთ ყოველთვის ჩართული VPN-ის დაყენება?"</string>
diff --git a/res/values-kk/strings.xml b/res/values-kk/strings.xml
index bf8fe2b43e..d7750475c2 100644
--- a/res/values-kk/strings.xml
+++ b/res/values-kk/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> профайлына жалғау"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Осы VPN желісі ажыратылсын ба?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Ажырату"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Нұсқасы: <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Нұсқа"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN ұмыту"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Бар VPN қолданбасын ауыстыру керек пе?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Әрқашан қосулы VPN режимін орнату"</string>
diff --git a/res/values-km/strings.xml b/res/values-km/strings.xml
index 4c47902c14..a42bff525c 100644
--- a/res/values-km/strings.xml
+++ b/res/values-km/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"តភ្ជាប់ <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ផ្តាច់ VPN នេះ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ផ្ដាច់"</string>
-    <string name="vpn_version" msgid="41856553718304367">"កំណែ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"កំណែ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"បំភ្លេច VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ជំនួស VPN ដែលមានស្រាប់ឬ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"កំណត់ VPN ឲ្យបើកជានិច្ចឬ?"</string>
diff --git a/res/values-kn/strings.xml b/res/values-kn/strings.xml
index 89bb43145d..7a995d1204 100644
--- a/res/values-kn/strings.xml
+++ b/res/values-kn/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> ಗೆ ಸಂಪರ್ಕಿಸಿ"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ಈ VPN ಸಂಪರ್ಕ ಕಡಿತಗೊಳಿಸುವುದೇ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ಸಂಪರ್ಕ ಕಡಿತಗೊಳಿಸಿ"</string>
-    <string name="vpn_version" msgid="41856553718304367">"<xliff:g id="VERSION">%s</xliff:g> ಆವೃತ್ತಿ"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ಆವೃತ್ತಿ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN ಮರೆತುಬಿಡು"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ಅಸ್ತಿತ್ವದಲ್ಲಿರುವ VPN ಸ್ಥಾನಾಂತರಿಸುವುದೇ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"VPN ಯಾವಾಗಲೂ ಆನ್ ಆಗಿರುವಂತೆ ಹೊಂದಿಸುವುದೇ?"</string>
diff --git a/res/values-ko/strings.xml b/res/values-ko/strings.xml
index 9f4947ce1e..de295c5eb3 100644
--- a/res/values-ko/strings.xml
+++ b/res/values-ko/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g>에 연결"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"이 VPN을 연결 해제하시겠습니까?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"연결 해제"</string>
-    <string name="vpn_version" msgid="41856553718304367">"버전 <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"버전"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN 삭제"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"기존 VPN을 교체하시겠습니까?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"연결 유지 VPN을 설정하시겠습니까?"</string>
diff --git a/res/values-ky/strings.xml b/res/values-ky/strings.xml
index a0547b7a6f..fde998aa05 100644
--- a/res/values-ky/strings.xml
+++ b/res/values-ky/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> менен байланышуу"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Бул VPN ажыратылсынбы?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Ажыратуу"</string>
-    <string name="vpn_version" msgid="41856553718304367">"<xliff:g id="VERSION">%s</xliff:g> версиясы"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Версия"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN профили унутулсун"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Учурдагы VPN алмаштырылсынбы?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Туруктуу VPN\'ди жөндөйсүзбү?"</string>
diff --git a/res/values-lo/strings.xml b/res/values-lo/strings.xml
index d68c9aee7c..545b2cc03e 100644
--- a/res/values-lo/strings.xml
+++ b/res/values-lo/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"ເຊື່ອມຕໍ່ຫາ <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ຕັດການເຊື່ອມຕໍ່ VPN ນີ້ບໍ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ຕັດການເຊື່ອມຕໍ່"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ເວີ​ຊັນ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ເວີຊັນ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"ລືມ VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ຂຽນທັບ VPN ທີ່ມີຢູ່ກ່ອນແລ້ວຂອງທ່ານບໍ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Set always-on VPN?"</string>
diff --git a/res/values-lt/strings.xml b/res/values-lt/strings.xml
index b3607fecd6..afaac38fd4 100644
--- a/res/values-lt/strings.xml
+++ b/res/values-lt/strings.xml
@@ -3309,7 +3309,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Prisijungti prie <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Atsijungti nuo šio VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Atsijungti"</string>
-    <string name="vpn_version" msgid="41856553718304367">"<xliff:g id="VERSION">%s</xliff:g> versija"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versija"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Pamiršti VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Pakeisti esamą VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Nustatyti visada įjungtą VPN?"</string>
diff --git a/res/values-lv/strings.xml b/res/values-lv/strings.xml
index 088b640e29..f7fc316d03 100644
--- a/res/values-lv/strings.xml
+++ b/res/values-lv/strings.xml
@@ -3269,7 +3269,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Savienojuma izveide ar <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Vai pārtraukt šo VPN savienojumu?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Pārtraukt savienojumu"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versija <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versija"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Aizmirst VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Vai aizstāt esošo VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Vai iestatīt vienmēr ieslēgtu VPN?"</string>
diff --git a/res/values-mk/strings.xml b/res/values-mk/strings.xml
index 816769370a..067a9a9d55 100644
--- a/res/values-mk/strings.xml
+++ b/res/values-mk/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Поврзи се со <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Да се исклучи оваа VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Прекини врска"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Верзија <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Верзија"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Заборави VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Да се замени постојната VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Да се постави „секогаш вклучена“ VPN?"</string>
diff --git a/res/values-ml/strings.xml b/res/values-ml/strings.xml
index f96e44cee1..e300229ab1 100644
--- a/res/values-ml/strings.xml
+++ b/res/values-ml/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> എന്നതിലേക്ക് കണക്റ്റുചെയ്യുക"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ഈ VPN വിച്ഛേദിക്കണോ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"വിച്ഛേദിക്കുക"</string>
-    <string name="vpn_version" msgid="41856553718304367">"പതിപ്പ് <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"പതിപ്പ്"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN മറക്കുക"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"നിലവിലുള്ള VPN-ന് പകരം പുതിയതാക്കണോ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"\'എല്ലായ്പ്പോഴും ഓൺ\' VPN സജ്ജമാക്കണോ?"</string>
diff --git a/res/values-mn/strings.xml b/res/values-mn/strings.xml
index dcf8544c8d..cba7087611 100644
--- a/res/values-mn/strings.xml
+++ b/res/values-mn/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> руу холбогдох"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Энэ VPN-г салгах уу?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Салгах"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Хувилбар <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Хувилбар"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN-г мартах"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Энэ VPN-г солих уу?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"VPN-г тогтмол асаалттайгаар тохируулах уу?"</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 3b9f320812..90df7a1f6d 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -3230,7 +3230,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> शी कनेक्ट करा"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"हे VPN डिस्कनेक्ट करायचे?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"‍डिस्कनेक्ट करा"</string>
-    <string name="vpn_version" msgid="41856553718304367">"आवृत्ती <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"आवृत्ती"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN विसरा"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"विद्यमान VPN पुनर्स्थित करायचे?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"नेहमी-सुरू VPN सेट करायचे?"</string>
diff --git a/res/values-ms/strings.xml b/res/values-ms/strings.xml
index bedb34e4f1..28cb6bc6ef 100644
--- a/res/values-ms/strings.xml
+++ b/res/values-ms/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Sambung ke <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Putuskan sambungan VPN ini?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Putuskan sambungan"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versi <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versi"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Lupakan VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Gantikan VPN yang sedia ada?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Tetapkan VPN sentiasa hidup?"</string>
diff --git a/res/values-my/strings.xml b/res/values-my/strings.xml
index 3d48d63f35..ef9167583d 100644
--- a/res/values-my/strings.xml
+++ b/res/values-my/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> နှင့် ချိတ်ဆက်ခြင်း"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ဤ VPN အား ချိတ်ဆက်မှု ဖြုတ်လိုပါသလား။"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ဆက်သွယ်မှု ဖြတ်ရန်"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ဗားရှင်း <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ဗားရှင်း"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN ကိုမေ့လိုက်ပါ"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ရှိပြီးသား VPN ကိုအစားထိုးမလား။"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"VPN ကို အမြဲတမ်းဖွင့်ထားရန် သတ်မှတ်မလား။"</string>
diff --git a/res/values-nb/strings.xml b/res/values-nb/strings.xml
index 15314c9c6f..ccd9357690 100644
--- a/res/values-nb/strings.xml
+++ b/res/values-nb/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Koble til <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Vil du koble fra dette VPN-et?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Koble fra"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versjon <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versjon"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Glem VPN-profilen"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Vil du erstatte det nåværende VPN-et?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Vil du angi alltid på-VPN?"</string>
diff --git a/res/values-ne/strings.xml b/res/values-ne/strings.xml
index 3d359067bc..b72330c8fd 100644
--- a/res/values-ne/strings.xml
+++ b/res/values-ne/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> मा जोड्नुहोस्"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"यो VPN को जडान विच्छेद गर्ने हो?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"डिस्कनेक्ट गर्नुहोस्"</string>
-    <string name="vpn_version" msgid="41856553718304367">"संस्करण <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"संस्करण"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN लाई बिर्सनुहोस्"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"विद्यमान VPN लाई बदल्ने हो?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"सधैँ-सक्रिय VPN लाई सेट गर्ने हो?"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 2df67bbeb9..c79520c739 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Verbinding maken met <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Verbinding met dit VPN verbreken?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Verbinding verbreken"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versie <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versie"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN vergeten"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Bestaande VPN vervangen?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Always-on VPN instellen?"</string>
diff --git a/res/values-or/strings.xml b/res/values-or/strings.xml
index 56070e3b20..da17a3e132 100644
--- a/res/values-or/strings.xml
+++ b/res/values-or/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> ସହ ସଂଯୋଗ କରନ୍ତୁ"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ଏହି VPNକୁ ବିଚ୍ଛିନ୍ନ କରିବେ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ବିଚ୍ଛିନ୍ନ କରନ୍ତୁ"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ସଂସ୍କରଣ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ସଂସ୍କରଣ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPNକୁ ଭୁଲିଯାଅ"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ପୂର୍ବରୁ ଥିବା VPNକୁ ବଦଳାଇବେ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ସର୍ବଦା-ଅନ୍‌ ଥିବା VPN ସେଟ୍‌ କରିବେ?"</string>
diff --git a/res/values-pa/strings.xml b/res/values-pa/strings.xml
index 2afd8d12c5..bdd2ed2441 100644
--- a/res/values-pa/strings.xml
+++ b/res/values-pa/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> ਨਾਲ ਕਨੈਕਟ ਕਰੋ"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ਇਹ VPN ਡਿਸਕਨੈਕਟ ਕਰੀਏ?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ਡਿਸਕਨੈਕਟ ਕਰੋ"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ਵਰਜਨ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ਵਰਜਨ"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN ਨੂੰ ਭੁੱਲ ਜਾਓ"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ਕੀ ਮੌਜੂਦਾ VPN ਨੂੰ ਤਬਦੀਲ ਕਰਨਾ ਹੈ?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ਕੀ ਹਮੇਸ਼ਾ-ਚਾਲੂ VPN ਨੂੰ ਸੈੱਟ ਕਰਨਾ ਹੈ?"</string>
diff --git a/res/values-pl/strings.xml b/res/values-pl/strings.xml
index c7c15d91f9..ded9665725 100644
--- a/res/values-pl/strings.xml
+++ b/res/values-pl/strings.xml
@@ -3309,7 +3309,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Połącz z: <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Odłączyć tę sieć VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Odłącz"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Wersja <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Wersja"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zapomnij VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Zastąpić obecną sieć VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Ustawić stały VPN?"</string>
diff --git a/res/values-pt-rBR/strings.xml b/res/values-pt-rBR/strings.xml
index e76fbe5854..e6258eb74d 100644
--- a/res/values-pt-rBR/strings.xml
+++ b/res/values-pt-rBR/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectar a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Desconectar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconectar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versão <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versão"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Esquecer VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Substituir VPN já existente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Configurar VPN sempre ativa?"</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index 7bc69a123e..5ef61c9688 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Ligar a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Pretende desligar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desligar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versão <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versão"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Esquecer VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Pretende substituir a VPN existente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Pretende definir a VPN como sempre ativa?"</string>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index e76fbe5854..e6258eb74d 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectar a <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Desconectar esta VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Desconectar"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versão <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versão"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Esquecer VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Substituir VPN já existente?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Configurar VPN sempre ativa?"</string>
diff --git a/res/values-ro/strings.xml b/res/values-ro/strings.xml
index 5b8dc3bcdd..d775643de1 100644
--- a/res/values-ro/strings.xml
+++ b/res/values-ro/strings.xml
@@ -3271,7 +3271,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Conectați-vă la <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Doriți să vă deconectați de la această rețea VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Deconectați-vă"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versiunea <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versiune"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Eliminați profilul VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Înlocuiți rețeaua VPN existentă?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Setați rețeaua VPN ca activată permanent?"</string>
diff --git a/res/values-ru/strings.xml b/res/values-ru/strings.xml
index 7b3212b3cc..4be0bebfba 100644
--- a/res/values-ru/strings.xml
+++ b/res/values-ru/strings.xml
@@ -3309,7 +3309,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Подключение к <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Отключиться от VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Да"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Версия <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Версия"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Удалить VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Сменить текущую VPN-сеть?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Настроить постоянную VPN-сеть?"</string>
diff --git a/res/values-si/strings.xml b/res/values-si/strings.xml
index d005bdf6aa..6f465f845d 100644
--- a/res/values-si/strings.xml
+++ b/res/values-si/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> වෙත සම්බන්ධ කරන්න"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"මෙම VPN එක විසන්ධි කරන්නද?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"විසන්ධි කරන්න"</string>
-    <string name="vpn_version" msgid="41856553718304367">"අනුවාදය <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"අනුවාදය"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN අමතක කරන්න"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"පවතින VPN ප්‍රතිස්ථාපනය කරන්නද?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"සැම විට ක්‍රියාත්මක VPN සකසන්නද?"</string>
diff --git a/res/values-sk/strings.xml b/res/values-sk/strings.xml
index 0dec38adb4..dcd8c860eb 100644
--- a/res/values-sk/strings.xml
+++ b/res/values-sk/strings.xml
@@ -3311,7 +3311,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Pripojiť k profilu <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Odpojiť tento profil VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Odpojiť"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Verzia <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Verzia"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Zabudnúť profil VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Nahradiť existujúcu sieť VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Nastaviť sieť VPN, ktorá je vždy zapnutá?"</string>
diff --git a/res/values-sl/strings.xml b/res/values-sl/strings.xml
index 0751036386..1c4c9aa34a 100644
--- a/res/values-sl/strings.xml
+++ b/res/values-sl/strings.xml
@@ -3311,7 +3311,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Povezovanje z <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Prekinitev povezave s tem omrežjem VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Prekini povezavo"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Različica <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Različica"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Pozabi omrežje VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Želite zamenjati obstoječe omrežje VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Ali želite, da je omrežje VPN stalno vklopljeno?"</string>
diff --git a/res/values-sq/strings.xml b/res/values-sq/strings.xml
index 9e32889aa3..e0126e6a88 100644
--- a/res/values-sq/strings.xml
+++ b/res/values-sq/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Lidhu me <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Shkëpute këtë rrjet VPN"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Shkëpute"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Versioni <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versioni"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Harroje rrjetin VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Të zëvendësohet rrjeti ekzistues VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Të caktohet rrjeti VPN gjithmonë aktiv?"</string>
diff --git a/res/values-sr/strings.xml b/res/values-sr/strings.xml
index b79cc59671..c26c2a5cbf 100644
--- a/res/values-sr/strings.xml
+++ b/res/values-sr/strings.xml
@@ -3269,7 +3269,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Повезивање на профил <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Желите ли да прекинете везу са овим VPN-ом?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Прекини везу"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Верзија <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Верзија"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Заборави VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Желите ли да замените постојећи VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Желите ли да подесите увек укључени VPN?"</string>
diff --git a/res/values-sv/strings.xml b/res/values-sv/strings.xml
index 26deb01a1d..f4bb685251 100644
--- a/res/values-sv/strings.xml
+++ b/res/values-sv/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Anslut till <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Vill du koppla från VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Koppla från"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Version <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Version"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Glöm VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Vill du byta ut befintligt VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Vill du ange Always-on VPN?"</string>
diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 7489038043..88e1d41633 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Unganisha kwa <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Ungependa kuondoa VPN hii?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Kata muungnisho"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Toleo la <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Toleo"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Ondoa VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Ungependa kubadilisha VPN iliyopo?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Ungependa kuweka VPN iliyowashwa kila mara?"</string>
diff --git a/res/values-ta/strings.xml b/res/values-ta/strings.xml
index a2800eb153..4c17df3b7c 100644
--- a/res/values-ta/strings.xml
+++ b/res/values-ta/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> உடன் இணை"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"VPNஐத் துண்டிக்கவா?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"தொடர்பைத் துண்டி"</string>
-    <string name="vpn_version" msgid="41856553718304367">"பதிப்பு <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"பதிப்பு"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPNஐ நீக்கு"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"தற்போதுள்ள VPNஐ மாற்றியமைக்கவா?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"எப்போதும் இயங்கும் VPNஐ அமைக்கவா?"</string>
diff --git a/res/values-te/strings.xml b/res/values-te/strings.xml
index 69a9bdcb4d..53b7059b61 100644
--- a/res/values-te/strings.xml
+++ b/res/values-te/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g>కి కనెక్ట్ చేయండి"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ఈ VPNను డిస్‌కనెక్ట్ చేయాలా?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"డిస్‌కనెక్ట్ చేయి"</string>
-    <string name="vpn_version" msgid="41856553718304367">"వెర్షన్ <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"వెర్షన్"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPNని విస్మరించు"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"ఇప్పటికే ఉన్న VPNని భర్తీ చేయాలా?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ఎల్లప్పుడూ ఆన్‌లో ఉండే VPNని సెట్ చేయాలా?"</string>
diff --git a/res/values-th/strings.xml b/res/values-th/strings.xml
index 81356a56e0..5233648e33 100644
--- a/res/values-th/strings.xml
+++ b/res/values-th/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"เชื่อมต่อกับ <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"ยกเลิกการเชื่อมต่อ VPN นี้ไหม"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"ยกเลิกการเชื่อมต่อ"</string>
-    <string name="vpn_version" msgid="41856553718304367">"เวอร์ชัน <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"รุ่น"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"ลืม VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"แทนที่ VPN ที่มีอยู่หรือไม่"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"ตั้งค่า VPN แบบเปิดตลอดเวลาหรือไม่"</string>
diff --git a/res/values-tl/strings.xml b/res/values-tl/strings.xml
index c59e9f4c29..e8d05dbde8 100644
--- a/res/values-tl/strings.xml
+++ b/res/values-tl/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Kumonekta sa <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Putulin ang koneksyon ng VPN na ito?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Putulin ang koneksyon"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Bersyon <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Bersyon"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Kalimutan ang VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Gusto mo bang palitan ang dati nang VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Gusto mo bang itakda ang VPN na palaging naka-on?"</string>
diff --git a/res/values-tr/strings.xml b/res/values-tr/strings.xml
index b41bec2d6c..9eee2b8a2d 100644
--- a/res/values-tr/strings.xml
+++ b/res/values-tr/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> profiline bağlan"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Bu VPN\'nin bağlantısı kesilsin mi?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Bağlantıyı kes"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Sürüm <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Sürüm"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPN\'yi unut"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Mevcut VPN değiştirilsin mi?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Her zaman açık VPN ayarlansın mı?"</string>
diff --git a/res/values-uk/strings.xml b/res/values-uk/strings.xml
index 2424b098cf..a03c73e287 100644
--- a/res/values-uk/strings.xml
+++ b/res/values-uk/strings.xml
@@ -3309,7 +3309,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Під’єднатися до профілю <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Відключити цю мережу VPN?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Відключити"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Версія <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Версія"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Забути мережу VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Замінити наявну мережу VPN?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Увімкнути постійну мережу VPN?"</string>
diff --git a/res/values-ur/strings.xml b/res/values-ur/strings.xml
index cf9791788f..a044e999a4 100644
--- a/res/values-ur/strings.xml
+++ b/res/values-ur/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> سے مربوط کریں"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"‏اس VPN سے غیر منسلک کریں؟"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"غیر منسلک کریں"</string>
-    <string name="vpn_version" msgid="41856553718304367">"ورژن <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"ورژن"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"‏VPN بھول جائیں"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"‏موجودہ VPN بدلیں؟"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"‏ہمیشہ آن VPN سیٹ کریں؟"</string>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index b09da08890..caa6bab543 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"<xliff:g id="PROFILE">%s</xliff:g> tarmog‘iga ulanish"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Bu VPN bilan aloqa uzilsinmi?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Aloqani uzish"</string>
-    <string name="vpn_version" msgid="41856553718304367">"<xliff:g id="VERSION">%s</xliff:g> versiya"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Versiya"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"VPNni o‘chirish"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Mavjud VPN tarmog‘i almashtirilsinmi?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Doimiy VPN o‘rnatilsinmi?"</string>
diff --git a/res/values-vi/strings.xml b/res/values-vi/strings.xml
index d3dd173504..53448e2186 100644
--- a/res/values-vi/strings.xml
+++ b/res/values-vi/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Kết nối với <xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Ngắt kết nối VPN này?"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Ngắt kết nối"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Phiên bản <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Phiên bản"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Xóa VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Thay thế VPN hiện có?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Đặt VPN luôn bật?"</string>
diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index 898845c658..e12a39ec00 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -3229,7 +3229,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"连接到<xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"要断开与此 VPN 的连接吗？"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"断开连接"</string>
-    <string name="vpn_version" msgid="41856553718304367">"版本 <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"版本"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"取消保存 VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"要替换现有 VPN 吗？"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"要设置始终开启的 VPN 吗？"</string>
diff --git a/res/values-zh-rHK/strings.xml b/res/values-zh-rHK/strings.xml
index 1bb9b080b2..83fc706a07 100644
--- a/res/values-zh-rHK/strings.xml
+++ b/res/values-zh-rHK/strings.xml
@@ -3232,7 +3232,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"連接「<xliff:g id="PROFILE">%s</xliff:g>」"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"要解除連結此 VPN 嗎？"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"中斷連線"</string>
-    <string name="vpn_version" msgid="41856553718304367">"版本為：<xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"版本"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"刪除 VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"要取代目前的 VPN 嗎？"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"要設定保持開啟的 VPN 嗎？"</string>
diff --git a/res/values-zh-rTW/strings.xml b/res/values-zh-rTW/strings.xml
index af7320832b..cacce52aa5 100644
--- a/res/values-zh-rTW/strings.xml
+++ b/res/values-zh-rTW/strings.xml
@@ -3231,7 +3231,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"連線到「<xliff:g id="PROFILE">%s</xliff:g>」"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"要中斷這個 VPN 連線嗎？"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"中斷連線"</string>
-    <string name="vpn_version" msgid="41856553718304367">"版本 <xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"版本"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"清除 VPN 設定檔"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"要取代現有的 VPN 嗎？"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"要設定永久連線的 VPN 嗎？"</string>
diff --git a/res/values-zu/strings.xml b/res/values-zu/strings.xml
index b74a120a67..1bef65254a 100644
--- a/res/values-zu/strings.xml
+++ b/res/values-zu/strings.xml
@@ -3230,7 +3230,7 @@
     <string name="vpn_connect_to" msgid="216709261691085594">"Xhuma ku-<xliff:g id="PROFILE">%s</xliff:g>"</string>
     <string name="vpn_disconnect_confirm" msgid="6356789348816854539">"Nqamula le-VPN"</string>
     <string name="vpn_disconnect" msgid="7753808961085867345">"Nqamula"</string>
-    <string name="vpn_version" msgid="41856553718304367">"Inguqulo engu-<xliff:g id="VERSION">%s</xliff:g>"</string>
+    <string name="vpn_version" msgid="6344167191984400976">"Uhlobo"</string>
     <string name="vpn_forget_long" msgid="729132269203870205">"Khohlwa i-VPN"</string>
     <string name="vpn_replace_vpn_title" msgid="3994226561866340280">"Shintshanisa i-VPN ekhona?"</string>
     <string name="vpn_set_vpn_title" msgid="1667539483005810682">"Setha i-VPN ehlala ivuliwe?"</string>
-- 
2.17.1

